neon_llm_core/chatbot.py

```diff
@@ -138,21 +142,23 @@ def _get_llm_api_response(self, shout: str) -> Optional[LLMProposeResponse]:
                                         request_data=request_data.model_dump(),
                                         target_queue=queue,
                                         response_queue=f"{queue}.response")
-            return LLMProposeResponse(**resp_data)
+            return LLMProposeResponse.model_validate(resp_data)
```

Comment:
To be consistent with other methods:
```suggestion
            return LLMProposeResponse.model_validate(obj=resp_data)
```

Reply:
fixed that